platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PayloadCfgSimple.java

```diff
@@ -160,10 +167,21 @@ private static class ClassVersion {
      */
     private float ratioOfFCMTransaction;
 
-    public PayloadCfgSimple() {}
+    public PayloadCfgSimple() {
+        //        logger.info(
+        //                LOGM_DEMO_INFO,
+        //                "PayloadCfgSimple.<init>, immutable: {}, StackTrace: {}",
+        //                isImmutable(),
+        //                StackTrace.getStackTrace());
+    }
 
     private PayloadCfgSimple(final PayloadCfgSimple sourcePayload) {
         super(sourcePayload);
+        //        logger.info(
+        //                LOGM_DEMO_INFO,
+        //                "PayloadCfgSimple.<init>(COPY), immutable: {}, StackTrace: {}",
+        //                isImmutable(),
+        //                StackTrace.getStackTrace());
```

Comment:
Please remove the commented-out code (or please uncomment it if it's useful.)

Reply:
Removed